logging/targets.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logging
+
+import (
+	"fmt"
+	"os"
+	"slices"
+	"strings"
+)
+
+const (
+	targetStdoutName = "STDOUT"
+	targetStderrName = "STDERR"
+)
+
+var targetNames = []string{
+	targetStdoutName,
+	targetStderrName,
+}
+
+// TargetNames returns the list of all log target names.
+func TargetNames() []string {
+	return slices.Clone(targetNames)
+}
```

Comment:
I don't think you need this anymore, since you don't enumerate the target options in the `default` case (since default is to fallback to file)